learing_journal/scripts/initializedb.py

```diff
@@ -32,16 +33,23 @@ def main(argv=sys.argv):
     options = parse_vars(argv[2:])
     setup_logging(config_uri)
     settings = get_appsettings(config_uri, options=options)
-
+    # settings['sqlalchemy.url'] = os.environ['DATABASE_URL']
     engine = get_engine(settings)
+    Base.metadata.drop_all(engine)
     Base.metadata.create_all(engine)
 
     # The below lines populates the database initialy
 
-    session_factory = get_session_factory(engine)
-
-    with transaction.manager:
-        dbsession = get_tm_session(session_factory, transaction.manager)
-        for entry in ENTRIES:
-            row = MyModel(title=entry['title'], body=entry['body'], date=entry['date'], id=entry['id'])
-            dbsession.add(row)
+    # session_factory = get_session_factory(engine)
+    #
+    # with transaction.manager:
+    #     dbsession = get_tm_session(session_factory, transaction.manager)
+    #     for entry in ENTRIES:
+    #         # intdented like this so the linter stopped complaining
+    #         row = Journal(title=entry['title'], body=entry['body'],
+    #                       date=datetime.datetime.strptime(entry['date'],
+    #                                                       '%B %d, %Y'),
+    #                       date_last_updated=datetime.datetime.strptime(
+    #                       entry['date'], '%B %d, %Y'),
+    #                       id=entry['id'])
+    #         dbsession.add(row)
```

Comment:
please bury dead code.  Don't leave things you aren't using in your submitted code.